front/src/modules/ui/data/view-bar/components/MultipleFiltersDropdownContent.tsx

```diff
@@ -34,8 +38,10 @@ export const MultipleFiltersDropdownContent = () => {
     ViewBarRecoilScopeContext,
   );
 
+  const dropdownWidth = useRecoilValue(dropdownWidthState);
+
   return (
-    <StyledDropdownMenu>
+    <StyledDropdownMenu width={dropdownWidth}>
```

Comment:
This has been renamed as DropdownMenu, could you rebase your PR on main?

Reply:
yes, i've rebased my pr